packages/sanity/src/desk/panes/document/documentPanel/documentViews/styles.ts

```diff
@@ -0,0 +1,21 @@
+import {Box, Heading} from '@sanity/ui'
+import styled, {css} from 'styled-components'
+
+export const Root = styled(Box)`
+  position: relative;
+`
+
+export const Title = styled(Heading)`
+  ${({theme}) => {
+    const fontSizeSmall = theme.sanity.fonts.heading.sizes[3].fontSize
+    const fontSizeLarge = theme.sanity.fonts.heading.sizes[4].fontSize
+
+    return css`
+      overflow-wrap: break-word;
```

Comment:
Changed from `word-break: break-word;` as break-word is [now deprecated](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#break-word)